src/pages/index.js

```diff
@@ -0,0 +1,304 @@
+import "../pages/index.css";
+import {
+  enableValidation,
+  settings,
+  disableButton,
+  resetValidation,
+} from "../scripts/validation.js";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c78896b4-8f8f-4aab-b711-572c50f5bd05",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardEl = getCardElement(item);
+      cardList.append(cardEl);
+    });
+
+    const avatarElement = document.querySelector(".profile__avatar");
+    profileName.textContent = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+    document.querySelector(".profile__avatar").src = userInfo.avatar;
```

Comment:
`document.querySelector(".profile__avatar")`


You've already found it in line 27 as `avatarElement`